src/main/java/httpserver/http/request/RequestHandler.java

```diff
@@ -1,20 +1,20 @@
-package httpServer;
+package httpserver.http.request;
 
 import java.io.BufferedReader;
 import java.io.IOException;
 
-class RequestHandler {
+public class RequestHandler {
     private final BufferedReader input;
 
     public RequestHandler(BufferedReader input) {
         this.input = input;
     }
 
-    String readResponse() {
+    public String readResponse() {
```

Comment:
I like the handler approach, but wonder if this is a typo?

Reply:
Thanks for spotting this, Ashley. I'll change it